src/app/common/dialogs/export-pdf-dialog/export-pdf-dialog.component.ts

```diff
@@ -3,13 +3,14 @@ import { Component, ElementRef, Renderer2, ViewChild, AfterViewInit } from '@ang
 import { BaseDialog } from '../base-dialog';
 import { RecipientBadgeInstance } from '../../../recipient/models/recipient-badge.model';
 
-import { jsPDF } from 'jspdf';
+import jsPDF from 'jspdf';
```

Comment:
I think we typically do use the curly braces, so we should probably stick to it to stay consistent

Reply:
using curly braces again